macros/cross-adapter-modeling/base/segment_web_page_views.sql

```diff
@@ -32,7 +32,11 @@ renamed as (
         search as page_url_query,
         
         referrer,
-        {{ dbt_utils.get_url_host('referrer') }} as referrer_host,
+        replace(
+            {{ dbt_utils.get_url_host('referrer') }},
```

Comment:
Hmm I'm wondering if we should update `get_url_host` to remove the `www.` part, rather than doing it here. Going to check internally.

Reply:
Ok so no one followed up, so I'm happy to do this trimming here. In a later version of utils we can strip the `www.` out of the URL in the `get_url_host` macro.